src/test/resources/data/incoming_requests/mocked_request1_with_sample_status.json

```diff
@@ -0,0 +1,258 @@
+{
+  "requestId": "MOCKREQUEST1_B",
+  "recipe": "GENESET101_BAITS",
+  "projectManagerName": "Bar, Foo",
+  "piEmail": "request1pi@mskcc.org",
+  "labHeadName": "Foo Bar",
+  "labHeadEmail": "request1pi@mskcc.org",
+  "investigatorName": "John Smith",
+  "investigatorEmail": "smithj@mskcc.org",
+  "dataAnalystName": "Poin Dexter",
+  "dataAnalystEmail": "dexterp@mskcc.org",
+  "otherContactEmails": "dexterp@mskcc.org",
+  "dataAccessEmails": "",
+  "qcAccessEmails": "",
+  "isCmoRequest": true,
+  "bicAnalysis": false,
+  "samples": [
+    {
+      "cmoSampleName": "C-MP789JR-X001-d",
+      "sampleName": "XXX002_P3_12345_L1",
+      "cmoSampleClass": "Primary",
+      "oncoTreeCode": "CLL",
+      "collectionYear": "",
+      "tubeId": "",
+      "qcReports": [],
+      "libraries": [
+        {
+          "barcodeId": "IDT29",
+          "barcodeIndex": "ATTGAGGA",
+          "libraryIgoId": "MOCKREQUEST1_B_1_1_1_1",
+          "libraryVolume": 35.0,
+          "libraryConcentrationNgul": 34.8,
+          "captureConcentrationNm": "11.49425287356322",
+          "captureInputNg": "400.0",
+          "captureName": "Pool-MOCKREQUEST1_B-Tube7_1",
+          "runs": [
+            {
+              "runMode": "HiSeq High Output",
+              "runId": "RUNID_0123",
+              "flowCellId": "X5KL2KKAY",
+              "readLength": "",
+              "runDate": "2018-06-05",
+              "flowCellLanes": [
+                5
+              ],
+              "fastqs": [
+                "/FASTQ/Project_MOCKREQUEST1_B/Sample_XXX002_P3_12345_L1_IGO_MOCKREQUEST1_B_1/XXX002_P3_12345_L1_IGO_MOCKREQUEST1_B_1_S82_R1_001.fastq.gz",
+                "/FASTQ/Project_MOCKREQUEST1_B/Sample_XXX002_P3_12345_L1_IGO_MOCKREQUEST1_B_1/XXX002_P3_12345_L1_IGO_MOCKREQUEST1_B_1_S82_R2_001.fastq.gz"
+              ]
+            }
+          ]
+        }
+      ],
+      "cmoPatientId": "C-MP789JR",
+      "igoId": "MOCKREQUEST1_B_1",
+      "investigatorSampleId": "XXX002_P3_12345_L1",
+      "species": "Human",
+      "sex": "F",
+      "tumorOrNormal": "Tumor",
+      "preservation": "Frozen",
+      "specimenType": "PDX",
+      "sampleOrigin": "Tissue",
+      "tissueLocation": "",
+      "baitSet": "GENESET101_BAITS",
+      "igoComplete": true,
+      "cmoSampleIdFields": {
+        "naToExtract": "",
+        "sampleType": "Tissue",
+        "normalizedPatientId": "MRN_REDACTED",
+        "recipe": "GENESET101_BAITS"
+      },
+      "status": [
+        {
+        "validationStatus": true,
+        "validationReport": ""
+        }
```

Comment:
mock data inside of `/incoming_requests` is modeled after what we would be receiving directly from igo so in this case `status` shouldn't exist on these samples